src/components/schedule-page/table/schedule/schedule.component.tsx

```diff
@@ -22,15 +22,17 @@ export function ScheduleComponent({
 }: ScheduleComponentOptions): JSX.Element {
   const scheduleLogic = useContext(ScheduleLogicContext);
   function isPresent(): boolean {
-    if (scheduleLocalState.isInitialized) {
+    if (scheduleLocalState.isInitialized && !scheduleLocalState.isAutoGenerated) {
```

Comment:
Z tego fragmentu nie jestem dumny. Na razie jakoś działa, ale trzeba to będzie poprawić. Z tym, że to już w innym zadaniu